packages/kbn-esql-editor/src/fetch_fields_from_esql.ts

```diff
@@ -26,16 +26,20 @@ export function fetchFieldsFromESQL(
   expressions: ExpressionsStart,
   time?: TimeRange,
   abortController?: AbortController,
-  dataView?: DataView
+  timeFieldName?: string
```

Comment:
This function is also called by the ESQL editor and the solutions. By changing the arg here, it means you need to change it to all the consumers too

Reply:
i was assuming that any place that calls this will be found by typescript check (which is passing)

Reply:
The one in the editor is not going to complain due to its particularity, The other in solutions I am not sure why it doesnt because I havent written. The fact is that needs to get updated everywhere

Reply:
the particularity of the editor one is that it does not pass in dataview (which is optional), which continues to work now when the optional parameter is time field name. other relevant places were already updated (alerting solution)